Resources/Prototypes/_Sunrise/Loadouts/loadout_groups.yml

```
@@ -227,6 +227,18 @@
   - WeaponDisabler
   - WeaponTaser
 
+# Security Cadet
+- type: loadoutGroup
+  id: SunriseWeaponCadet
+  name: loadout-group-security-gun
+  minLimit: 1
+  maxLimit: 1
+  loadouts:
+  - CadetWeaponGlockBox
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 3

(indentation)